### PR TITLE
additions into combine graph

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

File: instat/sdgCombineGraphOptions.vb [hunk 1]
```diff
@@ -15,22 +15,79 @@
...
     End Sub
 
     Public Sub SetDefaults()
-
+        ucrInputLeft.ResetText()
+        ucrInputRight.ResetText()
+        ucrInputTop.ResetText()
+        ucrInputBottom.ResetText()
+        nudColumns.Value = 2
+        nudRows.Value = 2
```
Comment (dannyparsons) [inline instat/sdgCombineGraphOptions.vb]: These are not the defaults, they come from the calculation in what Beth sent.

File: instat/sdgCombineGraphOptions.vb [hunk 2]
```diff
@@ -15,22 +15,79 @@
...
 
+    Private Sub nudRows_ValueChanged(sender As Object, e As EventArgs) Handles nudRows.ValueChanged
+        If nudRows.Text <> "" Then
+            clsRsyntax.AddParameter("nrow", nudRows.Value)
+        Else
+            clsRsyntax.RemoveParameter("nrow")
+        End If
+    End Sub
+
+    Private Sub nudColumns_ValueChanged(sender As Object, e As EventArgs) Handles nudColumns.ValueChanged
```
Comment (dannyparsons) [inline instat/sdgCombineGraphOptions.vb]: We use TextChanged instead of ValueChanged for nuds.

**Comment by dannyparsons** [comment]
This looks ok to accept for now as a basis.

**Comment by dannyparsons** [comment]
But still needs some more work.

**Comment by maxwellfundi** [comment]
Yeah, sure. it still needs some more work.

Regards
Maxwell Fundi
|| Phone :+254728508241||

**Pull request merged**